### DISCUSSION TITLE
有禁止拖拽进度的功能吗

### DISCUSSION THREAD

**Discussion by kong-ve** [bytedance/xgplayer#824] [Q&A] [answered]
**您使用的西瓜播放器版本是多少？ What version of xgplayer are you using?**

**您使用的操作系统和浏览器分别是？ What OS and browser are you using?**

**如何复现问题？ What did you do?**

**您期望的播放器正常行为是？ What did you expect to happen?**

**实际播放器的表现是？ What actually happened?**

**可填写您所在的公司和相关产品业务，方便我们提供更好的技术支持 You can write your company and product which uses xgplayer,  for helping us provide better technical support.**

**Comment by uxiew** [comment]
// 禁止拖动进度条
    disableProgress?: boolean

**Comment by yiwen03** [comment, marked as answer]
可以配置disableProgress: true
